content/docs/getting-started/deploy-app/deploy-cli.md

```diff
@@ -14,4 +14,147 @@ seo:
+  - You can access the APIs at `<IP_OF_CLUSTER_ORCHESTRATOR>:10000`
+  - You have read and familiarized yourself with the [API]({{< relref "deploy-api.md" >}}) and [Dashboard]({{< relref "deploy-dashboard.md" >}}) approaches.
+{{< /callout >}}
+
+
+## The OAK CLI
+The benefits of using this CLI are:
+- Easy installation via pip
+- Native Interface for the Oakestra APIs
+  - Eliminates the need to use external third-party tools
+- Accelerated & Simpler Workflows
+  - Removes the need to memorize necessary API endpoints
+  - The CLI commands can be chained together and used in custom scripts
+  - Automates tedious tasks away
+    - E.g. Acquiring the Login Token
+  
+
+
+## CLI Setup
+You can install the CLI via `pip install oak-cli`. <br>
+When you run the CLI for the first time it will welcome you.
+
+![OAK CLI Initial Welcome ASCII Art](./cli-images/welcome-message.png)
+
+
+### Configurating the CLI
+The OAK CLI supports different scenarios based on the concrete Oakestra use case and user preferences.
+
+This CLI initially hides many of its commands to avoid overwhelming new users.
+Additionally, not every user needs every available command, and not every command makes sense in every situation or environment.
+
+To configure the set of shown commands run `oak configuration local-machine-purpose`.
+
+{{< callout context="note" title="FYI - Using Command Shortcuts" icon="outline/bolt" >}}
+  Most `oak` commands have aliases that enable shorter commands and easier combinations, thus faster workflows.
+
+  E.g. Instead of typing out `oak configuration local-machine-purpose`.
+  Simply run `oak c l`.
+
+  The available aliases are shown directly in the `-h` output.
+{{< /callout >}}
+
+The CLI will ask you a set of questions about your intended use and your target environment.
+Based on your Yes/No responses it will provide you with the matching set of commands.
+{{< asciinema key="cli_configuration_demo" poster="0:16" >}}
+
+This configuration is persistently stored on your machine.
+
+Configuring your OAK-CLI is optional yet highly recommended.
+You can always revert your configuration to its initial state.
+
+For further information about the CLI configuration see the **CLI Manuals**.
+
+
+## Basic CLI Usage
+The root command for the CLI is **oak**
+
+{{< callout context="note" title="Need Help?" icon="outline/info-circle" >}}
+  Every `oak` CLI command comes with its own help text to support your understanding.
+
+  Simply add `--help` or `-h` to any command to find out more.
+{{< /callout >}}
+
+
+### Working with Application SLAs
+As detailed in the [API Approach]({{< relref "deploy-api.md" >}}) Oakestra apps and services are defined in SLAs.<br>
+The OAK-CLI comes with a set of pre-defined default SLAs.<br>
+All available SLAs can be inspected via the `oak application sla` command.
+{{< asciinema key="cli_sla_inspect" poster="0:16" >}}
+
+All SLAs are stored in your home directory in the `oak_cli` folder.
```

Comment:
I would suggest to not flood the user with info, what if we just say `oak_cli/SLAs` and then simply show 

```
~/oak_cli
 ├── SLAs
 │   ├── blank_app_without_services.json
 │   ├── default_app_with_services.json
 │   └── mysla.json
```

omitting the rest of the files such as .init_flag etc.?

Reply:
Agreed - I will abstract away the other content